frontend/src/components/LeadRow.js

```diff
@@ -1,5 +1,36 @@
 import React from 'react';
+import { TableRow, TableCell, Box } from '@material-ui/core';
+import EditOutlinedIcon from '@material-ui/icons/EditOutlined';
+import DeleteOutlineOutlinedIcon from '@material-ui/icons/DeleteOutlineOutlined';
+import { useStyles } from './Home';
```

Comment:
I thought this approach was interesting. I never considered importing `useStyles` hooks between different modules, but it makes sense because all the styles for the refactored components are already in `Home.js`.

I haven't put much thought into how to organize styles for this project yet. Not sure if it makes more sense in the long run to consolidate styles in one file and reuse them in different modules or to individually style components and reuse those between modules.

Reply:
To be honest, I'm not sure this is the best approach, but it's definitely worth a conversation about future styling! In my experience it is typical to have styles  for example App folder with App or Index.js inside it and then some kind of App.styles or App.css file. However for this project, I'm not familiar with material-ui and how many styled components we are re-using.